docker/Dockerfile.oldev

```diff
@@ -0,0 +1,59 @@
+FROM olbase 
+
+# Expose Open Library, Infogami, and SOLR
+EXPOSE 80 7000 8983
+
+ENV LANG en_US.UTF-8
+
+# required for postgres
+ENV LC_ALL POSIX
```

Comment:
It seems like `ENV` variables are inherited ( https://github.com/moby/moby/issues/33242 ), so these 2 can probably be removed from `oldev`.